marklogic-data-hub/src/main/java/com/marklogic/hub/impl/HubProjectImpl.java

```diff
@@ -647,6 +647,11 @@ private void writeToStream(OutputStream out) {
             Stream.of("entities", "flows", "src" + File.separator + "main", "mappings", "step-definitions", "loadData", "gradle",
                 "gradlew", "gradlew.bat", "build.gradle", "gradle.properties").forEach(file ->{
                 File fileToZip = getProjectDir().resolve(file).toFile();
+                if (!fileToZip.exists()) {
+                    // should we add check if some missing files/folders are mandatory and give error info?
```

Comment:
I think that logging that a folder doesn't exist is sufficient.

Reply:
@hao1st what if there are gradle-<evn>.properties?

Reply:
The original requirement was to exclude gradle-*.properties, as there may be sensitive data in those that shouldn't be shared.